checkstyle/checkstyle.xml

```diff
@@ -126,7 +126,7 @@
     </module>
     <module name="NPathComplexity">
       <!-- default is 200 -->
-      <property name="max" value="500"/>
+      <property name="max" value="512"/>
```

Comment:
rather than increasing the max value please add a suppression for the `Serdes` class